src/router/stepper.ts

```diff
@@ -600,6 +600,7 @@ export const stepperRoutes: Array<StepperRouteConfig> = [
         completed: false,
         excludeFromMenu: true,
         stepCompleteOnEnter: routeNames.Exceptions,
+        routeResolver:removeBarriersRouteResolver

```

Comment:
going to need a route resolver on the `other-supporting-factors` step to accommodate AC 4.a. `Back` button click. Can set the values manually in the store for now.